### ISSUE TITLE
Duplicated symbols with HockeySDK-Source and CP 0.37 as framework

### ISSUE THREAD

**Issue by marcelofabri** [bitstadium/HockeySDK-iOS#176]
I tried updating my project to use the new `HockeySDK-Source` podspec, but I'm getting several build errors because of duplicated symbols.

I'm using CocoaPods 0.37.2 and `use_frameworks!`, as it's a Swift project.

**Comment by rggjan** [comment]
I can confirm, the same thing happens for me.

Symbols like "_OBJC_IVAR_$_BitAppVersionMetaInfo._date" in "libPods-...-HockeySDK.a(BITAppVersionMetaInfo.o)"

**Comment by anlinde** [comment]
We currently don't know why this happens, the effect only comes up when using `use_frameworks!`. If you use `HockeySDK` pod this works just fine. @alloy Do you have an idea by chance?

Is there any specific reason you are using the `HockeySDK-Source` pod?

**Comment by alloy** [comment]
Nope, sorry, I’ve been out of the loop for a while now. It would be best to file a ticket on the CocoaPods repo and provide a reproducible project for the devs to test.

**Comment by lukasspiess** [comment]
Sorry for the inconvenience everybody. The problem is fixed in this commit https://github.com/bitstadium/HockeySDK-iOS/commit/a918cb9d0775fcef86b31735956028c05543cd65 and we are looking to make this available publicly shortly.

**Comment by marcelofabri** [comment]
Actually, I was using `HockeySDK-Source` because when using `HockeySDK` I got another error (probably because it now ships a compiled static binary):

```
[!] The 'Pods-XXXX' target has transitive dependencies that include static binaries: (/Users/XXX/Pods/HockeySDK/HockeySDK-iOS/HockeySDK.embeddedframework/HockeySDK.framework)
```

**Comment by anlinde** [comment]
@marcelofabri We cannot reproduce this over here. Please fully remove the SDK from your Podfile, run `pod install` and then add it again. Maybe that helps.

**Comment by marcelofabri** [comment]
I think I know what is the issue. We have an internal component which imports `HockeySDK` and when I add this component to `Podfile`, I get that error. 

If I just add `HockeySDK`, `pod install` succeeds.

**Comment by anlinde** [comment]
The fixes have been merged and you should now be able to update the podspec to 3.7.0 without any more issues.

**Comment by marcelofabri** [comment]
Thanks for the quick fix! :beers:

**Comment by marcelofabri** [comment]
Actually, now the project compiles, but I'm unable to use certain features:

```objc
[[BITHockeyManager sharedHockeyManager].feedbackManager showFeedbackComposeView];
```

> Property 'feedbackManager' not found on object of type 'BITHockeyManager *'


I can submit a sample project, but it was really an empty project with this `Podfile`:

```ruby
use_frameworks!

target 'testehockey' do
	pod 'HockeySDK-Source'
end

target 'testehockeyTests' do

end

```

**Comment by anlinde** [comment]
I am sorry, but I can't get the `HockeySDK-Source` podspec work with frameworks at the moment. We also find another glitch in the `HockeySDK` podspec and will try to push it soon

**Comment by anlinde** [comment]
We just released an updated version 3.7.1:

1. `HockeySDK` podspec is binary and now finally works with the resource files.
2. `HockeySDK-Source` podspec is the same as in version 3.6.4 but with udpated source

Sorry for the trouble guys.
